src/components/Home/StationWeatherDetails.tsx

```diff
@@ -1,10 +1,25 @@
 import { ExportedWeatherData } from "@/types";
 import SvgInline from "@/components/Common/SvgInline";
 
+enum WeatherConditions {
+    HUMIDITY = "Humidity",
+    BAROMETER = "Barometer",
+    WIND = "Wind",
+    RAIN = "Rain",
+}
+
+enum Measurements {
+    PERCENTAGE = "%",
+    PRESSURE = "hPa",
+    SPEED = "km/h",
+    MILLIMETER = "mm",
+}
```

Comment:
My favorite enums. Do you think we could move them to a dedicated file? I do believe we are going to use them in different places.

Reply:
Good idea I'll move them out yep, thinking to create a file in types folder probably with some more generic types, enums etc